hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/devtools/sidecar/DtAttachedSidecar.kt

```diff
@@ -178,59 +89,63 @@ internal fun DtMinimizedSidecarWindowContent(
-                            exit = if (useAnimatedTransitions) fadeOut(tween(50)) else ExitTransition.None
-                        )
-                        .tag(Tag.ExpandMinimiseButton)
-                        .clickable { isExpandedChanged(true) }
-                        .padding(DtPadding.small)
-                        .animateContentSize(alignment = Alignment.TopCenter),
-                ) {
-                    DtComposeLogo(
-                        Modifier.size(DtSizes.largeLogoSize).padding(DtPadding.small),
-                        tint = animateReloadStatusColor(
-                            idleColor = Color.White,
-                            reloadingColor = DtColors.statusColorOrange2
-                        ).value
-                    )
-                    DtMinimisedReloadCounterStatusItem(
-                        modifier = Modifier.size(DtSizes.reloadCounterSize),
-                        showDefaultValue = !devToolsUseTransparency,
-                    )
-                }
-            } else {
-                // Expanded state - show the full UI
-                DtAttachedSidecarBody(Modifier.fillMaxSize()) { isExpandedChanged(false) }
-            }
+private fun DtSidecarReloadButton() {
+    if (!HotReloadEnvironment.gradleBuildContinuous) {
+        DtIconButton(
+            onClick = { OrchestrationMessage.RecompileRequest().sendAsync() },
+            tooltip = "Reload UI",
+            tag = Tag.ActionButton,
+        ) {
+            DtImage(
+                image = DtImages.Image.SYNC_ICON,
+                modifier = Modifier.size(DtSizes.iconSize),
+                tint = Color.White,
+            )
         }
     }
 }
+
+@Composable
+private fun DtSidecarResetButton() {
+    DtIconButton(
+        onClick = { OrchestrationMessage.CleanCompositionRequest().sendAsync() },
+        tooltip = "Reset UI",
+        tag = Tag.ActionButton,
+    ) {
+        DtImage(
+            image = DtImages.Image.DELETE_ICON,
+            modifier = Modifier.size(DtSizes.iconSize),
+            tint = Color.White,
+        )
+    }
+}
+
+@Composable
+private fun DtSidecarRestartButton() {
+    DtIconButton(
+        onClick = { OrchestrationMessage.RestartRequest().sendAsync() },
+        tooltip = "Restart the application",
+        tag = Tag.ActionButton,
+    ) {
+        DtImage(
+            image = DtImages.Image.RESTART_ICON,
+            modifier = Modifier.size(DtSizes.iconSize),
+            tint = Color.White,
+        )
+    }
+}
+
+@Composable
+private fun DtSidecarCloseButton() {
+    DtIconButton(
+        onClick = { OrchestrationMessage.ShutdownRequest("Requested by user through 'devtools'").sendAsync() },
+        tooltip = "Close the application",
+        tag = Tag.ActionButton,
+    ) {
+        DtImage(
+            image = DtImages.Image.CLOSE_ICON,
+            modifier = Modifier.size(DtSizes.iconSize),
+            tint = Color.White,
+        )
+    }
+}
```

Comment:
Read through this file and I felt really fast here: 
Its easy to understand how the UI looks and behaves here. 
Well done!